src/styles/sign.css

```diff
@@ -0,0 +1,186 @@
+@import url('https://fonts.googleapis.com/css2?family=Cormorant+SC:wght@300&family=Edu+TAS+Beginner:wght@400;500;600;700&family=Kanit:wght@700&family=Open+Sans:wght@300;400;500;600;700&family=Playfair+Display:wght@400;500;600;700&family=Quicksand:wght@300;400;500;600;700&family=Roboto:ital,wght@0,100;0,300;0,500;0,700;1,100;1,300;1,400;1,500;1,900&display=swap');
```

Comment:
you can download the font file and have it in the assets and then import it, instead of doing it like that